openedx/core/djangoapps/schedules/management/commands/tests/test_send_recurring_nudge.py

```diff
@@ -136,34 +145,90 @@ def test_site_config(self, org_list, exclude_orgs, expected_message_count, mock_
         for config in (limited_config, unlimited_config):
             ScheduleConfigFactory.create(site=config.site)
 
-        filtered_sched = ScheduleFactory.create(
+        ScheduleFactory.create(
             start=datetime.datetime(2017, 8, 2, 17, 44, 30, tzinfo=pytz.UTC),
             enrollment__course__org=filtered_org,
         )
-        unfiltered_scheds = [
+        for _ in range(2):
             ScheduleFactory.create(
                 start=datetime.datetime(2017, 8, 2, 17, 44, 30, tzinfo=pytz.UTC),
                 enrollment__course__org=unfiltered_org,
             )
-            for _ in range(2)
-        ]
-
-        print(filtered_sched.enrollment)
-        print(filtered_sched.enrollment.course)
-        print(filtered_sched.enrollment.course.org)
-        print(unfiltered_scheds[0].enrollment)
-        print(unfiltered_scheds[0].enrollment.course)
-        print(unfiltered_scheds[0].enrollment.course.org)
-        print(unfiltered_scheds[1].enrollment)
-        print(unfiltered_scheds[1].enrollment.course)
-        print(unfiltered_scheds[1].enrollment.course.org)
 
         test_time_str = serialize(datetime.datetime(2017, 8, 2, 17, tzinfo=pytz.UTC))
         with self.assertNumQueries(1):
             tasks.recurring_nudge_schedule_hour(
                 limited_config.site.id, 3, test_time_str, org_list=org_list, exclude_orgs=exclude_orgs,
             )
 
-        print(mock_schedule_send.mock_calls)
         self.assertEqual(mock_schedule_send.apply_async.call_count, expected_message_count)
         self.assertFalse(mock_ace.send.called)
+
+    @ddt.data(
+        (19, 1),
+        (20, 0),
+        (21, 0)
+    )
+    @ddt.unpack
+    @patch.object(tasks, 'ace')
+    @patch.object(tasks, '_recurring_nudge_schedule_send')
+    def test_multiple_enrollments(self, test_hour, messages_sent, mock_schedule_send, mock_ace):
+        user = UserFactory.create()
```

Comment:
FYI: `UserFactory()` is equivalent to `UserFactory.create()`.

Reply:
I know. One of them confuses pylint, and the other doesn't, but I forget which is which. I think `.create` is the not-confusing one, because then pylint doesn't assume that the returned object is a `*Factory`.